recipes/AffjaxPostNode/src/Main.purs

```diff
@@ -0,0 +1,51 @@
+module AffjaxPostNode.Main where
+
+import Data.Argonaut
+
+import Affjax as AX
+import Affjax.RequestBody (json)
+import Affjax.ResponseFormat as ResponseFormat
+import Data.Argonaut as J
+import Data.Either (Either(..))
+import Data.Maybe (Maybe(..))
+import Effect (Effect)
+import Effect.Aff (launchAff_)
+import Effect.Class.Console (log)
+import Prelude (Unit, bind, ($), (<>))
+
+type Post = { id     :: Maybe Int
+            , title  :: String
+            , body   :: String
+            , userId :: Int
+            }
+
+postToJson :: Post -> Json
+postToJson = encodeJson
+
+postFromJson :: Json -> Either JsonDecodeError Post
+postFromJson = decodeJson
+
+main :: Effect Unit
+main = launchAff_ $ do
+  let 
+    postdata :: Post
+    postdata =  { id    : Nothing -- note how this returns filled-in if service works
+                , title : "title"
+                , body  : "body"
+                , userId: 22 }
+
+    endpoint :: String
+    endpoint = "http://jsonplaceholder.typicode.com/posts"
+    
+  result <- AX.post ResponseFormat.json endpoint (Just $ json $ encodeJson postdata)
```

Comment:
Missed this in my first review, but why are you using `encodeJson postData` rather than `postToJson`?

Reply:
oh, good catch, i was actually going back and forth on including the synonyms or the type qualification or the explicit written `encode` and `decode`, i'll fix it now

Reply:
I think the `postToJson` way is better because it shows the type signatures.